tffconnect_backend/tffconnect/games/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from ..referees.models import Referees
+
+
+class Games(models.Model):
+    id = models.AutoField(primary_key=True)
+    referee_id = models.ForeignKey(Referees, on_delete=models.CASCADE)
```

Comment:
aslinda referee ye point ediyor, referee_id degil

Reply:
ama id döndürüyor primary key'i id olduğu için kolay anlamak açısından referee_id koydum adını